ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule.hs

```diff
@@ -20,18 +20,18 @@
 -- and once it fulfills the state's criteria, it yields control back to the scheduler,
```

Comment:
There still is a dangling `AdvertisedPoints` in the comment above

Reply:
Found only one. Done in 6a7a36fb1a94e90a9134172b9955531d7b291fc9.